broker-node/app/Clients/BrokerNode.php

```diff
@@ -428,6 +438,10 @@ private static function sendToHookNode(&$chunks, $request)
         self::$NodeMessenger->spamHookNodes($tx, $spammedNodes);  // remove this, temporary solution
 
         self::updateHookNodeDirectory($hookNodeUrl, "request_made");
+		
+		//store the event record for analysis
+		self::initEventRecord();
+        self::$ChunkEventsRecord->addChunkSentToHookNodeEvent($hooknode['ip_address']);
```

Comment:
This becomes `ChunkEvent::addChunkSentToHookNodeEvent($hooknode['ip_address']);` if you make that method static.